.github/workflows/build-images.yaml

```diff
@@ -32,12 +32,26 @@ jobs:
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
       - name: Set Authorino version
         id: authorino-version
+        run: |
+          echo "VERSION=latest" >> $GITHUB_ENV
```

Comment:
I suppose we'll read from `build.yaml` here, right?

Reply:
Yep, exactly !